test/e2e/framework/pod/wait.go

```diff
@@ -301,7 +302,7 @@ func WaitForMatchPodsCondition(c clientset.Interface, opts metav1.ListOptions, d
 				return fmt.Errorf("Unexpected error: %v", err)
 			}
 			if !done {
-				conditionNotMatch = append(conditionNotMatch, fmt.Sprintf("%s_%s(%s)", pod.Name, pod.Namespace, pod.UID))
+				conditionNotMatch = append(conditionNotMatch, GetPodDesc(pod.Name, pod.Namespace, pod.UID))
```

Comment:
see https://github.com/kubernetes/kubernetes/pull/89504#discussion_r398715368
It is has been changed to ```fmt.Sprintf("%s_%s(%s)"``` , but i want to update them according to your comment.Thanks.
 /cc @neolit123